stl/inc/numbers

```diff
@@ -0,0 +1,188 @@
+// numbers standard header (core)
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _NUMBERS_
+#define _NUMBERS_
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+#if !_HAS_CXX20
+#pragma message("The contents of <numbers> are available only with C++20 or later.")
+#else // ^^^ !_HAS_CXX20 / _HAS_CXX20 vvv
+#ifdef __cpp_lib_concepts
+#include <concepts>
+#endif
+
+#pragma pack(push, _CRT_PACKING)
+#pragma warning(push, _STL_WARNING_LEVEL)
+#pragma warning(disable : _STL_DISABLED_WARNINGS)
+_STL_DISABLE_CLANG_WARNINGS
+#pragma push_macro("new")
+#undef new
+
+_STD_BEGIN
+namespace numbers {

```

Comment:
Out of curriosity: Should `numbers` be uglified?

Reply:
Doesn't seem like it should.
https://github.com/microsoft/STL/blob/f9b1dccd121b5f8bb76d06adb0c85aac3f766326/stl/inc/chrono#L24-L25
https://github.com/microsoft/STL/blob/f9b1dccd121b5f8bb76d06adb0c85aac3f766326/stl/inc/ranges#L23-L24

Reply:
> Doesn't seem like it should.

Yep, we currently don't introduce such macros. (That might change in vNext, depending on how we decide to use inline namespaces.)